### PR TITLE
Sebd dev

### PR DESCRIPTION
**Pull request by Sebastian-D**

### CODE REVIEW

File: content/english/data_types/genomics_transcriptomics/services.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The [SciLifeLab Genomics Platform](https://www.scilifelab.se/infrastructure/?cat
...
 
-We believe this workflow may be useful also for COVID-19 projects aiming at understanding host susceptibility. Data generation is to 30x coverage using PCR-free libraries, bioinformatic analysis is comprehensive and includes calling of SNV, INDEL, CNV, SV, STR and UPDs. Variants are processed using a ranking model prior to made available for interpretation in Scout. This infrastructure has been applied to analysis of >6000 WGS samples.
+We believe this workflow may be useful also for COVID-19 projects aiming at understanding host susceptibility. Data generation is to 30x coverage using PCR-free liraries. The bioinformatic analysis is comprehensive and includes calling of SNV, INDEL, CNV, SV, STR and UPDs. Variants are processed using a ranking model prior to being made available for interpretation in Scout. This infrastructure has been applied to analysis of >6000 WGS samples.
```
Comment (pekrau) [inline content/english/data_types/genomics_transcriptomics/services.md:19]: "liraries"?

Comment (pekrau) [reply content/english/data_types/genomics_transcriptomics/services.md:19]: How do I suggest an actual edit?

File: content/english/data_types/protein_data/data.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ menu:
 
 ## Published Data
 
-The [Human Protein Atlas](https://www.proteinatlas.org) provides open access to protein expression data across human organs, tissues and cells. This protein resource is also organized into special sections focussing on the tissue expression, subcellular expression, proteins in the brain or in blood, proteins related to pathology and metabolic pathways.
+The [Human Protein Atlas](https://www.proteinatlas.org) provides open access to protein expression data across human organs, tissues and cells. This protein resource is organized into focused sections for tissue expression, subcellular expression, proteins in the brain or in blood, proteins related to pathology and metabolic pathways.
```
Comment (ewels) [inline content/english/data_types/protein_data/data.md:13]: Learn something new every day! https://en.wiktionary.org/wiki/focussed 😅

**Comment by ewels** [comment]
@pekrau - see https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request

> Optionally, to suggest a specific change to the line or lines, click , then edit the text within the suggestion block.


Then @Sebastian-D can easily merge them from the pull-request interface: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes

File: content/english/data_types/genomics_transcriptomics/services.md (hunk 1 above)
Comment (Sebastian-D) [reply content/english/data_types/genomics_transcriptomics/services.md]: Dunno. I just went to the diff and chose to edit the file and then made the change.

Comment (ewels) [reply content/english/data_types/genomics_transcriptomics/services.md]: https://github.com/ScilifelabDataCentre/covid-portal/pull/99#issuecomment-637542640

Comment (ewels) [reply content/english/data_types/genomics_transcriptomics/services.md]: I can't demo this now, as the line has changed. But this is the button that you're looking for ☝️ 

image

File: content/english/data_types/protein_data/data.md (hunk 2 above)
Comment (Sebastian-D) [reply content/english/data_types/protein_data/data.md:13]: Wow, I had no idea! Luckily, this is a "prestige publication" so we are definitely going with "focused"

Comment (ewels) [reply content/english/data_types/protein_data/data.md:13]:
```suggestion
The [Human Protein Atlas](https://www.proteinatlas.org) provides open access to protein expression data across human organs, tissues and cells. This protein resource is organized into focussed sections for tissue expression, subcellular expression, proteins in the brain or in blood, proteins related to pathology and metabolic pathways.
```

Just a demo for @pekrau and @Sebastian-D of a suggested change in a pull request.

No need to accept it 😉

**Comment by Sebastian-D** [comment]
I think I made the change that @pekrau requested. Do I suggest he re-review? I don't think I can merge with triage permission.

**Comment by ewels** [comment]
Interesting - usually a review is marked as old once there are new commits. Not sure why that didn't happen here. I have dismissed it 👍

**Pull request merged**